src/services/files/settings/types.ts

```diff
@@ -0,0 +1,38 @@
+import { AffectedRecords, OptionsBase } from '../../types';
+
+export interface FileServiceSettingsService {
```

Comment:
Uggo name?

Reply:
`FileSettingsService` would be consistent with the other files 🤷